demo/mxcube-web/server.yaml

```diff
@@ -9,6 +9,9 @@ server:
   DEBUG: false
 
   ALLOWED_CORS_ORIGINS: []
+  USER_DB_PATH: /tmp/mxcube_users.db
```

Comment:
I think this (the database path related things) should go into the other PR, right ?